docs/source/user-guide/cli/installation.md

```diff
@@ -56,8 +56,7 @@ this to work.
 
 ```bash
 git clone https://github.com/apache/datafusion
-cd arrow-datafusion
-git checkout 12.0.0
+cd datafusion
 docker build -f datafusion-cli/Dockerfile . --tag datafusion-cli
 docker run -it -v $(your_data_location):/data datafusion-cli
```

Comment:
I'd propose to add `--rm` here  and take out `-v $(your_data_location):/data`.
this would result in the command being directly copy-pastable into terminal without modifications and also would reduce side effects.

Reply:
Maybe we could add a comment like

```shell
# Run with data mounted at /data in the container
# docker run --rm -it -v $(your_data_location):/data datafusion-cli

# Run in a container without access to the host filesystem
docker run --rm -it  datafusion-cli
```

Reply:
🤔  though the second command doesn't work

```shell
andrewlamb@Andrews-MacBook-Pro-2 datafusion % docker run --rm -it  datafusion-cli
error: Invalid value "/data" for '--data-path <DATA_PATH>': Invalid data directory '/data'
```